ntoskrnl/mm/mmfault.c

```diff
@@ -252,16 +257,25 @@ MmAccessFault(IN ULONG FaultCode,
         return MmArmAccessFault(FaultCode, Address, Mode, TrapInformation);
     }
 
+Retry:
```

Comment:
I think this retry block is small enough that it would be better readable to have a while loop here.

Reply:
I don't think that makes it any more readable. It's just an infinite loop with an if/break.